xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-test/xwiki-platform-flamingo-skin-test-docker/src/test/it/org/xwiki/flamingo/test/docker/DeletePageIT.java

```diff
@@ -374,4 +380,54 @@ public void deletePageWithUsedClass(TestUtils setup, TestInfo info)
         ViewPage viewPage = setup.gotoPage(freeReference);
         assertFalse(viewPage.exists());
     }
+
+    /**
+     * This test checks the behaviour of the page deletion form when the user is allowed to choose between sending the
+     * page to the recycle bin or delete it permanently.
+     * In this test the user chooses to send the page to the recycle bin.
+     */
+    @Test
+    @Order(9)
+    public void deleteToRecycleBin(TestUtils setup)
+    {
+        // Set the user type to Advanced
+        Map<String, Object> userProperties = new HashMap<>();
+        userProperties.put("usertype", "Advanced");
+        setup.updateObject("XWiki", "superadmin", "XWiki.XWikiUsers", 0, userProperties);
+
+        // Set the canSkipRecycleBin property to true
```

Comment:
It seems there's a mix of names. I see:
* `isAllowedToSkipTheRecycleBin()` (see above)
* And now `canSkipRecycleBin`

I'm proposing to use a common terminology... So either `canSkipRecycleBin` everywhere or `isAllowedToSkipTheRecycleBin` everywhere. WDYT?

Reply:
`canSkipRecycleBin` and `isAllowedToSkipTheRecycleBin` have slightly different meanings:
- `canSkipRecycleBin` is the configuration that defines if the "skip the recycle bin feature" is activated.
- `isAllowedToSkipTheRecycleBin` tells if a given user is allow to skip the recycle bin (ie, the user is advanced and canSkipRecycleBin is true).

Reply:
Then `canSkipRecycleBin` is a bit wrong since it depends on whether the user is allowed to skip or not and `can` means whether it can be done or not.

Maybe:
* `isRecycleBinSkippingActivated()` (or `isRecycleBinSkippingConfigured`) since you said yourself: "defines if the "skip the recycle bin feature" is activated."
* `isRecycleBinSkippingAllowed()` (which is nicer than `isAllowedToSkipTheRecycleBin`  IMO, the `the` is not very nice)

I find that more consistent. WDYT?

Reply:
I agree. Updating.